api/overview/azure/eventgrid.md

```diff
@@ -1,116 +1,38 @@
 ---
 title: Azure Event Grid libraries for .NET
 description: Reference for Azure Event Grid libraries for .NET
-ms.date: 04/16/2018
+ms.date: 07/20/2021
 ms.topic: reference
 ms.service: event-grid
 ---
 
 # Azure Event Grid libraries for .NET
-
 Build event-driven applications that listen and react to events from Azure services and custom sources using simple HTTP-based event handling with Azure Event Grid.
 
-[Learn more](/azure/event-grid/overview) about Azure Event Grid and get started with the [Azure Blob storage event tutorial](/azure/storage/blobs/storage-blob-event-quickstart-powershell). 
+[Learn more](/azure/event-grid/overview) about Azure Event Grid and get started with the [Azure Blob storage event tutorial](/azure/event-grid/blob-event-quickstart-portal). 
 
 ## Client SDK
 
-Create events, authenticate, and post to topics using the Azure Event Grid Client SDK.
+Create events, authenticate, and post to topics using the Azure Event Grid client SDK.
 
-Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.EventGrid) directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
+Install the [NuGet package](https://www.nuget.org/packages/Azure.Messaging.EventGrid/) directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
 
 #### Visual Studio Package Manager
 
 ```powershell
-Install-Package Microsoft.Azure.EventGrid
+Install-Package Azure.Messaging.EventGrid
 ```
 
 #### .NET Core CLI
 
 ```dotnetcli
-dotnet add package Microsoft.Azure.EventGrid 
+dotnet add package Azure.Messaging.EventGrid
 ```
 
-### Publish events
+For an overview of the latest client SDK, samples, see the [Azure Event Grid client library for .NET](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Messaging.EventGrid_4.5.0/sdk/eventgrid/Azure.Messaging.EventGrid) page. This page also has a link to the migration guide that helps you migrate applications that use the old .NET library (**Microsoft.Azure.EventGrid**) to use the latest one (**Azure.Messaging.EventGrid**).
```

Comment:
Do we want to link to the branch itself here?  That's going to lock this to v4.5.0 and need to be updated as we release packages.   Should we use the `main` branch instead to keep it evergreen?

```
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventgrid/Azure.Messaging.EventGrid
```